src/tracing/instrumentation/kafka.js

```diff
@@ -108,37 +80,23 @@ function shimEmit(original) {
       return original.apply(this, arguments);
     }
 
-    var uid = hook.initAndPreSimulated();
-    hook.setTracingSuppressed(uid, false);
-    var spanId = tracingUtil.generateRandomSpanId();
-    var span = {
-      s: spanId,
-      t: spanId,
-      f: tracingUtil.getFrom(),
-      async: false,
-      error: false,
-      ec: 0,
-      ts: Date.now(),
-      d: 0,
-      n: 'kafka',
-      stack: [],
-      data: {
-        kafka: {
-          access: 'consume',
-          service: message.topic
-        }
+    cls.ns.runAndReturn(() => {
```

Comment:
We should avoid ES6 features for the sake of supporting Node.js 0.12 as mentioned above